### PR TITLE
[India] Added Data Portal India

### PR DESCRIPTION
**Pull request by yasarshaikh**
Provided Repository is not owned/associated by me however, it's associated by Govt. 

URL for DataPortalIndia
GitHub - https://github.com/DataPortalIndia 
Official Site - https://data.gov.in/  
Email - [ndsap@gov.in](mailto:ndsap@gov.in)
  
 :heart: GitHub Government Contributors

### CODE REVIEW

**Comment by mlinksva** [comment]
It looks like ONDC is a non-profit (section 8) organization rather than a government organization. I will merge the DataPortalIndia suggestion. Please feel free to correct me!

**Review by mlinksva — COMMENTED** [review]
sort, remove ONDC as not a government organization afaict

**Review by mlinksva — CHANGES_REQUESTED** [review]
sort, remove ONDC as not a government organization afaict

**Comment by yasarshaikh** [comment]
@mlinksva - Updated the file as suggested. Please check now and let me know if any other changes are required.

**Comment by mlinksva** [comment]
Thanks! Apologies for the noise above. I was trying to use the `.` code editor (first time user in context of a PR) to make changes but did not realize it could only (in this context) add to a review.

File: _data/governments.yml [hunk 1]
```diff
@@ -303,8 +303,10 @@ Hungary:
   - fphgov
 
 India:
+  - DataPortalIndia
   - ISRO
   - itschool
+
```
Comment (mlinksva) [inline _data/governments.yml]: Could you delete this extra line? Thanks!

**Review by mlinksva — CHANGES_REQUESTED** [review]

File: _data/governments.yml (hunk 1 above)
Comment (yasarshaikh) [reply _data/governments.yml]: Deleted, thanks!

**Pull request merged**